data/json/effects.json

```diff
@@ -2026,12 +2026,43 @@
   },
   {
     "type": "effect_type",
-    "id": "pkill1",
+    "id": "bleedrate_up",
+    "max_intensity": 3,
+    "rating": "bad",
+    "blood_analysis_description": "Blood Thinner",
+    "effect_dur_scaling": [ { "effect_id": "bleed", "modifier": 1.05, "same_bp": false } ]
+  },
+  {
+    "type": "effect_type",
+    "id": "ibuprofen_eff",
+    "//": "Blood pressure up.  Remember that this is divided by 100, so maximum 5% increase.",
```

Comment:
Actually, a bp_effect_mod of 500 would eventually correspond to a 500% increase, not a 5% increase. This is because, these values are divided by 100 indeed, but then when that value is added to the circulation_resistance, 500 / 100 = 5.0, which would result in bp_item_effect_mod being 5.0, and when added to the 1.0, it would result in 6.0, or 6x increase in blood pressure. Just wanted to point this out so as to avoid potential bugs in the future when high blood pressure will cause problems

Reply:
These are some things I pointed out. I would suggest adjusting the mod values of these effects in the JSON to fix this

Reply:
woops, you are right, I brain farted. I will fix this.